pkg/state/api.go

```diff
@@ -43,6 +43,7 @@ type StateInfo interface {
 	HeightToBlockID(height proto.Height) (proto.BlockID, error)
 	// FullWavesBalance returns complete Waves balance record.
 	FullWavesBalance(account proto.Recipient) (*proto.FullWavesBalance, error)
+	NewestFullWavesBalance(account proto.Recipient) (*proto.FullWavesBalance, error)
```

Comment:
`Newest` functions should not be added to this interface. They are not thread safe, they are only intended to be used by `ride` and `consensus` packages, not the node or api.